History/history_mod.F90

```diff
@@ -1796,7 +1795,7 @@ SUBROUTINE History_ReadCollectionData( Input_Opt,  State_Chm, State_Diag,  &
        DO C = 1, CollectionCount
           print*, 'Collection        ', TRIM( CollectionName       (C) )
           print*, '  -> FileName     ', TRIM( CollectionFileName   (C) )
-          print*, '  -> Format       ', TRIM( CollectionFormat     (C) )
+         !print*, '  -> Format       ', TRIM( CollectionFormat     (C) )
```

Comment:
Remove line rather than commenting out?

Reply:
Now removed in commit b349dde6208df7ce892cbd9f012ba5327a1083a5.  Actually, we no longer use the CollectionFormat metadata, so I've removed that array from `history_mod.F90`.